exercises/simple-linked-list/.meta/hints.md

```diff
@@ -0,0 +1,35 @@
+## Implementation Hints
+
+Do not use the struct `SimpleLinkedList` just as a wrapper around a `Vec`, but use the fields `head` and `len`.  
```

Comment:
- Consider phrasing as "Do not implement..." instead of "Do not use..."
- Possibly replace this sentence as follows:

> Do not implement the strict `SimpleLinkedList` as a wrapper around a `Vec`. Instead, allocate nodes on the heap.

Some students may want to omit the `len` field in favor of recursive computation on the fly, and we have no reason to forbid that design choice.